### PR TITLE
fix: expedite benchmark tests

### PR DESCRIPTION
**Pull request by noob20000405**
Signed-off-by: noob20000405 <vincent4869code@gmail.com>

Fix: https://github.com/casbin/casbin-cpp/issues/155

### Description

Commented those functions for benchmark tests which could cost a lot of time. E.g., 
```
BenchmarkCachedRBACModelMedium()
BenchmarkCachedRBACModelLarge()
...
```

### CODE REVIEW

File: tests/benchmarks/management_api_b.cpp [hunk 1]
```diff
@@ -74,7 +74,7 @@ static void BenchmarkHasPolicyMedium(benchmark::State& state) {
         params = { "user" + std::to_string(GetRandom1000()), "data" + std::to_string(GetRandom1000()/10), "read" }, e.HasPolicy(params);
 }
 
-BENCHMARK(BenchmarkHasPolicyMedium);
+// BENCHMARK(BenchmarkHasPolicyMedium);
```
Comment (sheny1xuan) [inline tests/benchmarks/management_api_b.cpp:77]: Maybe we need a `macro` to cover those code consume too much time, and makes `benchmark` is easy to test for users. What do you think? @noob20000405

Comment (hsluoyz) [reply tests/benchmarks/management_api_b.cpp:77]: @EmperorYP7

Comment (EmperorYP7) [reply tests/benchmarks/management_api_b.cpp:77]: We may also add a CMake option named `INTENSIVE_BENCHMARK` defaulting it to `OFF`. The user may turn it on through `-DINTENSIVE_BENCHMARK=ON` while generating the project files.

Comment (hsluoyz) [reply tests/benchmarks/management_api_b.cpp:77]: @noob20000405 can you add it?

Comment (noob20000405) [reply tests/benchmarks/management_api_b.cpp:77]: Sure! I'll do that right away.

Comment (noob20000405) [reply tests/benchmarks/management_api_b.cpp:77]: > @noob20000405 can you add it?

Sure! I'll do that right away.

**Comment by noob20000405** [comment]
Replaced by #162

**Pull request closed without merging**